cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -880,31 +854,40 @@ def test_service_dynamic_dialog_tagcontrol(appliance, import_datastore, import_d
     )
 
 
-@pytest.mark.meta(coverage=[1744413])
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_datepicker_in_service_request():
+@pytest.mark.customer_scenario
+@pytest.mark.meta(automates=[1744413])
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.parametrize("file_name", ["bz_1744413.yml"], ids=["load-init"])
+def test_datepicker_in_service_request(generic_catalog_item_with_imported_dialog):
     """
     Bugzilla:
         1744413
 
     Polarion:
         assignee: nansari
+        startsin: 5.11
         casecomponent: Services
         initialEstimate: 1/6h
-        startsin: 5.10
         testSteps:
-            1. Create Dialog with Date picker
-            2. Create Service with the dialog and order the service
-            3. Mention a date for the date picker
-            4. Navigate to Services -> Requests -> [your_service] -> Dialog Options
+            1. datepicker dialog
+            2. Create a service and add the dialog
+            3. Order the service
+            4. Go to on service request details page
         expectedResults:
             1.
             2.
             3.
-            4. Date picker should show correct selected dates
+            4. datepicker date should be correct
     """
-    pass
+    catalog_item, _, _ = generic_catalog_item_with_imported_dialog
+    request = ServiceCatalogs(
+        catalog_item.appliance, catalog=catalog_item.catalog, name=catalog_item.name
+    ).order()
+
+    view = navigate_to(request, "Details")
+
+    # Date should be correct on request details page
+    assert view.details.request_details.read()["Datepicker"] == "06/18/2020"
```

Comment:
Any way to pull the date from the dialog data? I.e. not have it hard-coded?

Reply:
@john-dupuy  in dialog data date format is different. `default_value: '2020-06-18T18:30:00.000Z'`

Reply:
I think you could do 
```python
data.split("T")[0].replace("-", "/")
```

Reply:
Hmm I think we could do some stuff with `datetime`, but I'd consider it _optional_ since this date won't change.

Reply:
yeah month and year are swapped